src/comments/interfaces/comments.service.interface.ts

```diff
@@ -0,0 +1,33 @@
+import { Comment } from '@prisma/client';
+import { CreateCommentDto } from '../dto/requests/create-comment.dto';
+import { UpdateCommentDto } from '../dto/requests/update-comment.dto';
+
+export interface ICommentsService {
+  createComment(
+    createCommentDto: CreateCommentDto,
+    userId: number,
+    postId: number,
+    groupId: number,
+  ): Promise<Comment>;
+
+  getCommentsByPost(
+    postId: number,
+    groupId: number,
+    parentId?: number,
+  ): Promise<(Comment & { user: { name: string } })[]>;
```

Comment:
1. repository에서 user의 id, name을 조회 및 응답함.
2. service에서 user의 id, name을 응답함
3. class-transfomer를 톨해 응답 객체를 생성함

이런 흐름인데요. service에서 user의 응답으로 id가 명시되지 않았습니다.
typescript 특성상 type은 런타임에서 사라지기 때문에 동작에 영향은 없지만 유지보수 및 디버깅을 위해 필요한 타입은 정확하게 명시되면 좋을거같습니다.

실제로 User 객체 전체를 응답하지만 id, name만 응답 타입으로 명시하는 경우는 문제가 없습니다만(실제 응답 데이터이지만 이런 데이터만 엑세스 가능하다를 표현하는 경우) 현재처럼 엑세스 할 수 없다는듯이 코드가 작성돼있지만 다른 계층(controller)에서 사용되는건 문제라고 생각합니다


```suggestion
  ): Promise<(Comment & { user: { id: number; name: string } })[]>;
```